additionalFiles/defaults/pipelines/stream-TwoStreamingNodesWithDifferentFlows.js

```diff
@@ -0,0 +1,107 @@
+const pipe = {
+    "name": "sub-ein-tst",
+    "kind": "stream",
+    "flowInput": {
+        "process_time": 0.02,
+        "flows": [
```

Comment:
Both flows have the save rate. The idea was to test different rates from each node